scripts/fuzzing.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -e
+source ./scripts/test_lib.sh
+
+# To fix according to newer version of go:
+# go get golang.org/dl/gotip
+# gotip download
+# GO_CMD="gotip"
+GO_CMD="go"
+
+log_callout -e "\\nExecuting v3rpc with target $FUZZ_TARGET \\n"
+
+cd $BASE_PATH
```

Comment:
This script requires providing multiple non-trivial environment variables for it to work. This script should only be called via github action, but also locally to reproduce issues. I think we should make the script independent so it can be called without to much hardship. 

Can we have the script include a good defaults for running? Or if the default cannot be provided, could we add validation to give better error message?

Reply:
I updated the script:

- FUZZ_TIME now defaults to 300s
- TARGET_PATH display an error in the case that is unset
- FUZZ_TARGET display an error in the case that is unset

Let me know if you prefer another setup